### DISCUSSION TITLE
Correspondance ECU

### DISCUSSION THREAD

**Discussion by Dralem57** [cedricp/ddt4all#1052] [Q&A] [answered]
Bonjour,

Je souhaite me connecter à ma Clio 3 pour résoudre un probléme de dillution d'huile.
Lorsque je lance le scan ECU j'ai le message suivant:
Engine Control Module (pas de correspondance parfaite) :DCM34 - P4BCx_X61_X77_X85_X44_V16 DIAGVERSION [76] SUPPLIER [153] SOFT [344B] VERSION [E000 instead C900]
Est ce que je n'ai pas la bonne dase de donnée ECU?

**Comment by KarelSvo** [comment]
everything is fine, take it

**Comment by Furtif** [comment]
> Bonjour,
> 
> Je souhaite me connecter à ma Clio 3 pour résoudre un probléme de dillution d'huile. Lorsque je lance le scan ECU j'ai le message suivant: Engine Control Module (pas de correspondance parfaite) :DCM34 - P4BCx_X61_X77_X85_X44_V16 DIAGVERSION [76] SUPPLIER [153] SOFT [344B] VERSION [E000 instead C900] Est ce que je n'ai pas la bonne dase de donnée ECU?

pas de correspondance parfaite mais je pense que l'ecu est le bon.

**Reply by Furtif** [reply]
vous pouvais toute fois utiliser une database récente disponible ici https://github.com/cedricp/ddt4all/discussions/845

ne dézippez pas le zip placez le dans le répertoire dtt4all .

**Reply by Furtif** [reply]
je vous recommande de faire une installation au propre https://github.com/cedricp/ddt4all/releases/tag/v3.0.1 désinstaller votre ancienne  version téléchargez la nouvelle ou utiliser le source repo directement puis placer le zip de dans. (sa évite conflits).
salutations.

**Reply by Dralem57** [reply]
Merci, je viens de refaire l’installation propre, j'ai trouvé la page avec la dillution d'huile.
Par contre je ne vois de champ pour renseigner les valeurs.
![pbddt4all](https://github.com/cedricp/ddt4all/assets/163517867/163c209e-f78e-4d3a-b3ee-487172b3684a)

**Reply by Furtif** [reply]
il vous faudra aller dans les ecrans edition 
![image](https://github.com/cedricp/ddt4all/assets/11375035/a1447e39-90d8-4504-8922-f0bb69e32aa1)
 celui que vous aviez mis est read (lecture) montre les infos seulement.

**Reply by Furtif** [reply, marked as answer]
voir quel est le bon pour vous Diesel ou essence sa je ne peut pas vous le dire.

**Comment by Dralem57** [comment]
Merci beaucoup!!!! j'ai réussi a résoudre mon probléme!

**Reply by julien133** [reply]
Salut,
Je pense avoir le même soucis de dilution d'huile sur ma Clio 3 avec code défaut P253F.
Peux tu me dire stp quels paramètres tu as rentré en écriture dans l'écran d'édition ? 

Autre point : J'ai un souci de connexion, ça lague énormément quand je charge le DCM34 et il n'arrive pas à lire les données, quelle référence de boitier ELM tu utilises ? connecté en wifi ?

Merci d'avance

**Reply by Dralem57** [reply]
![Uploading IMG_20240509_181302.jpg…]()

**Reply by Dralem57** [reply]
J'utilise ce boîtier

**Reply by julien133** [reply]
Merci pour ta réponse, mais je n'arrive pas à visualiser l'image de ton lien.
peux tu me donner les références stp?

**Reply by Dralem57** [reply]
https://www.amazon.fr/gp/aw/d/B08RJHFF2T?psc=1&ref=ppx_pop_mob_b_asin_title

**Reply by julien133** [reply]
Nikel merci c'est commandé !
Tu te rappel des paramètres que tu as mis en écriture pour supprimer le défaut de dilution d'huile de ta clio ?

**Reply by Dralem57** [reply]
La $211B

**Reply by julien133** [reply]
Salut, j'arrive enfin au menu pour écrire.
Par contre tu as écris quoi exactement dans la $211B ?
J'ai testé avec "0", "1", "3", rien n'y fait ça ne corrige pas le problème.

**Reply by julien133** [reply]
up

**Reply by Dralem57** [reply]
J'ai un doute maintenant, il me semble que le défaut revenu quand je modifié le 211. Je crois que j'ai modifié les 2218 et 2119 pour des valeurs plus proche de mon kilométrage actuelle. C'était à 65K alors que je suis à 330K.

**Reply by julien133** [reply]
Je vois que sur la mienne aussi c'était à 65k alors que j'ai 240k.
J'ai essayé d'écrire des valeurs proches de mon kilométrage mais rien n'y fait, quand je fait une lecture ça met une valeur qui n'a rien à voir.
Exemple je rentre en écriture 220k et quand je retourne en lecture il m'affiche 55k...
Je sais plus trop quoi faire de cette merguez...

**Comment by Furtif** [comment]
> Merci beaucoup!!!! j'ai réussi a résoudre mon probléme!

Cool 👍
